client/my-sites/sidebar-unified/menu.jsx

```diff
@@ -64,18 +64,24 @@ export const MySitesSidebarUnifiedMenu = ( {
 		<li>
 			<ExpandableSidebarMenu
 				onClick={ () => {
-					if ( link ) {
+					// @automattic/viewport doesn't have a breakpoint for medium (782px)
```

Comment:
I still think this `782` would be better represented as `$collapsedSidebarViewportWidth` or some such. Self documenting code style.

Reply:
Since `782` breakpoint is not something temporary, I added it in the viewport package and used `isWithinBreakpoint( '>782px' )` in order to align with the rest codebase.  acf3791 cb17657